src/track/trackmetadatataglib.cpp

```diff
@@ -1026,6 +1026,18 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
     const TagLib::ID3v2::FrameList bpmFrame(tag.frameListMap()["TBPM"]);
     if (!bpmFrame.isEmpty()) {
         parseBpm(pTrackMetadata, toQStringFirstNotEmpty(bpmFrame));
+        long bpmValue = pTrackMetadata->getBpm().getValue();
```

Comment:
getValue() returns a double, why is it passed to a long? 
In general we should avoid long, because it means different things on Windows a Linux.